### DISCUSSION TITLE
[ FF7 ] Fatal crash on entering Mako reactor (repeated)

### DISCUSSION THREAD

**Discussion by john-savepoint** [julianxhokaxhiu/FFNx#205] [Q&A]
**Describe the bug**
Game has major consistent crash when entering the Mako reactor at the start of the game. 

**To Reproduce**
It reoccurs when you enter the reactor from the big T walkway (similar to what you fight Airbuster on later) and is trying to load in the first room where Barret is starts talking about Mako and the planet.

**Expected behavior**
This is a total crash, expect it to play

**Screenshots**
[crash.zip](https://github.com/julianxhokaxhiu/FFNx/files/6031763/crash.zip)

**GPU (please complete the following information):**
- Brand : Intel UHD Graphics 620
- Driver Version :  27.20.100.8935

**Additional context**
Game says to post crash log here. I have attached in a zip file
Mods Installed:
-New Threat Mod v2.0.94
-Final Fantasy 7 Steam version 1.0.9
-Satsuki Yatoshi Upscale Mod Version 5.21 release

**Comment by julianxhokaxhiu** [comment]
No support for 7th Heaven will be provided here. Please use the relative dedicated Discord support channels.

**Reply by john-savepoint** [reply]
This isn't 7th heaven

**Reply by julianxhokaxhiu** [reply]
Are you using FFNx vanilla on Steam with no external tooling? Sorry, I saw you mentioned NT and Satsuki's and most of the time people come here because of a 7th Heaven crash.

**Comment by john-savepoint** [comment]
Well, I added some other stuff since first contacted which has completely broken the game but initially all I installed was Satsuki which seems to be bundled with FFNx. Because of this bundling I can't seem to find the FFNx log file actual. Other than that it was just stock steam and the new threat mod.

Not using 7th heaven for it.

IMPORTANT: I did figure out that it was the HD models in Satsuki that was causing the glitch that crash the game. When I turn this off I could get past it and get to the reactor core but I tried turning texture back on and going back to the same room the one with the two doors Jesse and Biggd open but the game the crashes again. I'm not sure if we God Scorpion model from New threat that is causing the crash as his model is not in the stock game

Get TypeApp for Android<http://www.typeapp.com/r?b=16417>
On 24 Feb 2021, at 9:44 am, Julian Xhokaxhiu <notifications@github.com<mailto:notifications@github.com>> wrote:

Are you using FFNx vanilla on Steam with no external tooling? Sorry, I saw you mentioned NT and Satsuki's and most of the time people come here because of a 7th Heaven crash.

**Comment by john-savepoint** [comment]
[FFNx.log](https://github.com/julianxhokaxhiu/FFNx/files/6032832/FFNx.log)
 I found the log file.

I have discovered that it is the HD Models from Satsuki that is causing the crash with New Threat. It is in the room that the Guard Scorpion is in that causes the crash.

**Reply by julianxhokaxhiu** [reply]
Glad you figured that out. Also the NT Author confirmed the same, most likely it's a clash of mods. Nothing related to FFNx.

**Comment by john-savepoint** [comment]
Sorry for delay, was busy playing your great work. Thank you.
________________________________

**Reply by julianxhokaxhiu** [reply]
Enjoy, that's the reason it was made for :)
